repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/mount_usr.sh

```diff
@@ -38,8 +36,14 @@ mount_usr()
                     _dev="/dev/disk/by-uuid/${_dev#UUID=}"
                     ;;
             esac
+
+            # TODO: -ef replaced with inode number comparsion - is this enough?
+            # TODO: supresses all shellcheck warnings regarding declared, but
+            # not assigned variables in the whole if statement - is this the
+            # right way of dealing with this warning?
+            # shellcheck disable=SC2154 # Variable root is assigned by dracut
```

Comment:
SC2154 is suppressed (not only in this case), because a variable is assigned by dracut, however this could suppress warnings about other variables within the same command scope.

Reply:
You can always introduce a new local variable e.g. `_root="${root#block:}"` and disable the warning only on this assignment.